runtime/lua/vim/_inspector.lua

```diff
@@ -177,13 +184,8 @@ function vim.show_pos(bufnr, row, col, filter)
   if #items.semantic_tokens > 0 then
     append('Semantic Tokens', 'Title')
     nl()
-    for _, token in ipairs(items.semantic_tokens) do
-      local client = vim.lsp.get_client_by_id(token.client_id)
-      client = client and (' (' .. client.name .. ')') or ''
-      item(token.hl_groups.type, 'type' .. client)
-      for _, modifier in ipairs(token.hl_groups.modifiers) do
-        item(modifier, 'modifier' .. client)
```

Comment:
is "modifier" not shown?

Reply:
It does display the modifiers. Here is a sample of the current output of `:Inspect`:

![inspect_output](https://user-images.githubusercontent.com/926528/220954105-506eb563-2ddb-4e27-afa7-fbd7afbed8d3.png)

I modified the inspector logic: instead of replicating the token-to-highlight logic from `semantic_tokens.lua`, it now simply finds all extmarks set by the semantic token engine (as determined by their namespace). In addition to avoiding repetition, this has the practical benefit of showing any non-default token highlights set by the user.

I also made a minor tweak to show the highlight priorities for the semantic tokens, which will hopefully reduce any future confusion about "why is my highlight not showing?"